src/Components/Web.JS/yarn.lock

```diff
@@ -346,16 +346,22 @@
     "@types/yargs" "^13.0.0"
 
 "@microsoft/dotnet-js-interop@link:../../JSInterop/Microsoft.JSInterop.JS/src":
-  version "0.0.0"
```

Comment:
@dotnet/aspnet-blazor-eng I assume we don't want these top three changes, and only want the ones after line 361 ?

Reply:
@mkArtakMSFT I saw all these PRs come in automagically. I'm not against taking them, but I think it would be a good opportunity for us to revise or JS generation. I've filed an issue [here](https://github.com/dotnet/aspnetcore/issues/29763)

Reply:
Hmm, should we configure Dependabot to ignore our three `@link` dependencies when changing yarn.lock files. Not sure that's possible given the fact Dependabot is supposedly only updating `bl` in this PR (for example). But, would it help❔

BTW why does Dependabot think these dependencies have version `5.0.0-dev` instead of `6.0.0-dev`❔

Reply:
@BrennanConroy do we need to fix files such as src/SignalR/clients/ts/signalr/package.json to get the versions right in `main`❔

Reply:
Right versions of what?

Reply:
1. `@microsoft/dotnet-js-interop`
2. `@microsoft/signalr-protocol-msgpack`
3. `@microsoft/signalr`

Reply:
I still have no idea what you're asking. What are you saying needs to be fixed in src/SignalR/clients/ts/signalr/package.json?

Reply:
My apologies. I'm trying to understand why Dependabot seems to think `5.0.0-dev` is the correct version of those packages. Noticed the package.json files in those projects have `"version": "5.0.0-dev"`. But, I also see correctly-versioned `*.tgz` after local or CI builds. Now I'm confused: What does `"version"` in package.json really control❔

Reply:
Oh now I see. I don't think that version matters very much. I believe it gets overwritten at build time. But I'll double check.

Reply:
@javiercn are you proposing to ignore these PRs for now, and wait for your work in the referenced issue?